src/main/java/bio/terra/cli/context/utils/Logger.java

```diff
@@ -0,0 +1,184 @@
+package bio.terra.cli.context.utils;
+
+import static ch.qos.logback.classic.Logger.ROOT_LOGGER_NAME;
+
+import bio.terra.cli.context.GlobalContext;
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.encoder.PatternLayoutEncoder;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.ConsoleAppender;
+import ch.qos.logback.core.OutputStreamAppender;
+import ch.qos.logback.core.filter.Filter;
+import ch.qos.logback.core.rolling.FixedWindowRollingPolicy;
+import ch.qos.logback.core.rolling.RollingFileAppender;
+import ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy;
+import ch.qos.logback.core.spi.FilterReply;
+import ch.qos.logback.core.util.FileSize;
+import ch.qos.logback.core.util.StatusPrinter;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.io.File;
+import org.slf4j.LoggerFactory;
+
+/** Utility methods for logging. */
```

Comment:
nit: this is more than just utility methods. Maybe re-phrase as "Utility class to initialize a Logback context for the main CLI app. Requires an already-loaded GlobalContext for defining logging file outputs and loading config properties."

Reply:
Updated.